sdk/src/wallet/account/operations/syncing/mod.rs

```diff
@@ -24,10 +24,24 @@ use crate::{
 };
 
 impl AccountHandle {
+    /// Set the fallback SyncOptions for account syncing
+    pub async fn set_default_sync_options(&self, options: SyncOptions) {
+        *self.fallback_sync_options.lock().await = options;
+    }
+
+    // Get the default sync options we use when none are provided
+    pub async fn sync_options(&self) -> SyncOptions {
```

Comment:
Name it `default_sync_options()` so it's aligned?

Reply:
i thought it was ugly :D

Reply:
Done